set.mm

```diff
@@ -52971,21 +52950,19 @@ of the restricted converse (see ~ cnvrescnv ).  (Contributed by NM,
       ( vx vy wceq cv wcel wa copab cxp eleq2 anbi2d opabbidv df-xp 3eqtr4g ) A
       BFZDGCHZEGZAHZIZDEJRSBHZIZDEJCAKCBKQUAUCDEQTUBRABSLMNDECAODECBOP $.
 
-    $( Membership in a Cartesian product.  Uses fewer axioms than ~ elxp .
-       (Contributed by NM, 4-Jul-1994.) $)
-    elxpi $p |- ( A e. ( B X. C ) -> E. x E. y ( A = <. x , y >. /\
-                ( x e. B /\ y e. C ) ) ) $=
-      ( vz vw cxp wcel cv cop wceq wa wex weq eqeq1 anbi1d 2exbidv copab cab
-      df-xp df-opab eqtri elab2gw ibi ) CDEHZICAJZBJZKZLZUGDIUHEIMZMZBNANZFJZUI
-      LZUKMZBNANZGJZUILZUKMZBNANUMFGCUFUFFGOZUPUTABVAUOUSUKUNURUIPQRURCLZUTULAB
-      VBUSUJUKURCUIPQRUFUKABSUQFTABDEUAUKABFUBUCUDUE $.
-
     $( Membership in a Cartesian product.  (Contributed by NM, 4-Jul-1994.) $)
     elxp $p |- ( A e. ( B X. C ) <-> E. x E. y ( A = <. x , y >. /\
                ( x e. B /\ y e. C ) ) ) $=
       ( cxp wcel cv wa copab cop wceq wex df-xp eleq2i elopab bitri ) CDEFZGCAH
       ZDGBHZEGIZABJZGCSTKLUAIBMAMRUBCABDENOUAABCPQ $.
 
+    $( Membership in a Cartesian product.  One direction of ~ elxp .
```

Comment:
This proof adds a dependency on ax-sep, ax-nul and ax-pr to the theorem (via opex, elopab (modified)).  Revert it.

Reply:
@icecream17 Can you in a separate PR prove the forward direction of elopab (label it elopabi, and relabel the current elopabi to, say, elopabes (for "explicit substitution") or elopabim), and then prove elopab and elxpi from it ?   The proof of that new elopabi is basically a subproof of the current elopab and elxpi, so it permits some "factoring".

Can you tell me you plan to do it by replying here, or else I'll do it.

Reply:
I'll do it